project/AppDependencies.scala

```diff
@@ -12,7 +12,7 @@ object AppDependencies {
     hmrc                %% s"domain-$playVersion"            % "9.0.0",
     hmrc                %% "stub-data-generator"             % "1.1.0",
     "org.typelevel"     %% "cats-core"                       % "2.10.0",
-    "ai.x"              %% "play-json-extensions"            % "0.42.0",
+    "ai.x"              %% "play-json-extensions"            % "0.42.0" exclude ("com.typesafe.play", "play-json_2.13"),
```

Comment:
do we know what this ai.x dependency is being used for? can it not be removed or replaced?